cmd/agent/main.go

```diff
@@ -107,6 +107,7 @@ type loggerArgs struct {
 	namespace        string
 	endpoint         string
 	component        string
+	headerAllowList  []string
```

Comment:
This should be a global configuration as well ?

Reply:
I think it is a platform config more than a user parameter, for global configuration it can be added here
https://github.com/kserve/kserve/blob/master/config/configmap/inferenceservice.yaml#L284